### ISSUE TITLE
Issues changing the distance threshold the getMappingPointsWithinMesh

### ISSUE THREAD

**Issue by Jsalas424** [openep/openep-core#70]
Hello,

I"m having issues changing the distance threshold for the getMappingPointsWithinMesh function.

Steps to reproduce:

```
>> openep_demo

>> iPoint = getMappingPointsWithinMesh( userdata, 'tol', 0.1 )

SWITCH expression must be a scalar or a character vector.

Error in pointStatus (line 43)
        switch varargin{i}

Error in getMappingPointsWithinMesh (line 39)
[iPoint, ~] = pointStatus( userdata, varargin );
 
>> iPoint = getMappingPointsWithinMesh( userdata, 'tol', 0.2 )
SWITCH expression must be a scalar or a character vector.

Error in pointStatus (line 43)
        switch varargin{i}

Error in getMappingPointsWithinMesh (line 39)
[iPoint, ~] = pointStatus( userdata, varargin );
```

Thanks!

**Comment by p-j-smith** [comment]
Hello, welcome to the openep repo!

Thanks for reporting the issue. I've just submitted a fix for it. There's a problem with the way `getMappingPointsWithinMesh` passes the arguments to `pointStatus`, which it calls underneath.

In the meantime, if you want to determine which points are inside your mesh with a different distance threshold, you could call `pointStatus` directly:

`[iPoint, ~] = pointStatus( userdata, 'tol', 0.5)`

**Comment by Jsalas424** [comment]
Clarifying question about this function:

1) What are the units for 'tol' ?
2) Does this work in addition to the distance threshold filter you declare during a case, or are all of the points (regardless of distance threshold) exported by CARTO and we're supposed to reprocess the data with this function?

**Comment by p-j-smith** [comment]
The units for `tol` should be the same as those for the mapping points (so mm). I'm not familiar with the filter set during a case (I'm a software developer and should not be poking catheters around inside someone's heart!), but as far as I'm aware Carto will indeed export all points and they will need to be filtered afterwards. But perhaps @drsewilliams know more

**Comment by p-j-smith** [comment]
This was fixed by #71
